diagnostic.md

```diff
@@ -5,74 +5,93 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+A backend is used to store updates made by the user via the front end and to serve up data to the user when she
+requests it from the front end.
```

Comment:
Good its the way we ensure our data persists when our session ends. It also allows multiple users to interact from different devices while using the same app.